src/github/query.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable camelcase */
+
+import { fetchJson, URL } from '../vendor/requests';
+import { Template } from '../vendor/Template';
+import { GMTDate as Date, ISO_8601 } from '../vendor/dates';
+import { selectFrom } from '../vendor/vectors';
+import { coalesce } from '../vendor/utils';
+
+// EXAMPLE https://github.com/mozilla-mobile/fenix/labels/eng%3Aperformance
+const GITHUB_ISSUES = new Template('https://api.github.com/repos/{{owner}}/{{repo}}/issues');
+
+const BURNDOWN = {
+  owner: 'mozilla-mobile',
+  repo: 'fenix',
+  labels: 'eng:performance',
+  filter: 'all',
+};
```

Comment:
@airimovici  I am not certain how complicated the queries to Github will get.  I put all the filter parameters in here for now.